.eslintrc.js

```diff
@@ -21,4 +21,17 @@ module.exports = {
 			],
 		},
 	},
+	rules: {
+		'jsdoc/check-tag-names': [
+			'warn',
+			{
+				definedTags: [
+					'values', // required for vue-styleguideist
```

Comment:
It is not required for styleguideist, it supports `@type` with string alternative `@type {'a'|'b'|'c'}`

Reply:
And we don't have `@values` anymore

Reply:
Fixed